Agents/CEAAgent/README.md

```diff
@@ -69,59 +69,48 @@ The CEA Agent is reachable at http://localhost:3838/cea-agent, where 3838 is the
-- ```graphName```: (optional) named graph to which the CEA triples belong to and will be instantiated under. If ```graphName``` is not specified, the agent will assume no graph usage.
+- ```iris```: array of building IRIs.
+- ```geometryEndpoint```: (optional) endpoint where the geospatial information of the buildings from ```iris``` is stored; if not specified, agent will default to querying building geometry from Ontop endpoint within the same stack.
+- ```usageEndpoint```: (optional) endpoint where the building usage information of the buildings from ```iris``` is stored, if not specified, agent will default to querying building usage from Ontop endpoint within the same stack.
+- ```ceaEndpoint```: (optional) endpoint where the CEA triples, i.e. energy demand and photovoltaic potential information, instantiated by the agent is to be stored; if not specified, agent will default to setting ```ceaEndpoint``` to the stack Blazegraph namespace, labelled by ```cea.label``` in [CEAAgentConfig.properties].
+- ```weatherEndpoint```: (optional) endpoint where historical weather information is stored; if not specified, agent will default to setting ```weatherEndpoint``` agent will default to setting ```ceaEndpoint``` to the stack Blazegraph namespace, labelled by ```cea.label``` in [CEAAgentConfig.properties].
+- ```terrainDatabase``` (optional) database from which the agent will attempt to query terrain data; if not specified, it will be set to the stack database specified as ```postgis.database``` in [CEAAgentConfig.properties].
+- ```terrainTable```: (optional) table, inclusive of schema prefix, from which the agent will attempt to query terrain data; if not specified, it will be set to the stack table specified as ```postgis.table``` in ```./cea-agent/src/main/resources/postgis.properties```.
 
 After receiving request to the run endpoint, the agent will query for the following CEA inputs from the endpoints specified in the request parameters: building geometry, surrounding buildings' geometry, building usage, historical weather data, and terrain data. The agent will then run CEA with the queried information as inputs, and send request with the CEA output data to the update endpoint afterwards. With the exception of building geometry query, whose results are necessary for the agent to run, if any of the CEA inputs queries fails, the agent will run CEA with its corresponding CEA default values for the missing inputs.
+Provide only the `iris` parameter in the request if agent is to query CEA inputs from the same stack as this agent with the default endpoints provided in [CEAAgentConfig.properties].
 
 Example request:
 ```
 { "iris" :
```

Comment:
It would be helpful/user-friendly to write out a full curl request command line that people can copy-paste, rather than just provide the `json` content.

Reply:
Added cURL.